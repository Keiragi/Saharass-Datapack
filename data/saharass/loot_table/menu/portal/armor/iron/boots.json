{
  "pools": [
    {
      "rolls": 1,
      "entries": [
        {
          "type": "item",
          "name": "iron_boots",
          "functions": [
            {
              "function": "set_name",
              "name": {
                "text": "鉄のブーツ",
                "color": "white"
              },
              "target": "item_name"
            },
            {
              "function": "set_attributes",
              "modifiers": []
            },
            {
              "function": "set_custom_data",
              "tag": "{\"Menu\":{\"Button\":\"armor/iron/boots\"}}"
            },
            {
              "function": "set_lore",
              "entity": "this",
              "mode": "replace_all",
              "lore": [
                {
                  "text": "保管数 : ",
                  "extra": [
                    {
                      "score": {
                        "objective": "ItemStorage",
                        "name": "$108"
                      },
                      "extra": [
                        {
                          "text": "個"
                        }
                      ]
                    }
                  ],
                  "color": "white",
                  "italic": false
                }
              ]
            },
            {
              "function": "reference",
              "name": "saharass:menu/not_yet",
              "conditions": [
                {
                  "condition": "value_check",
                  "value": {
                    "type": "score",
                    "target": {
                      "type": "fixed",
                      "name": "$108"
                    },
                    "score": "ItemStorage"
                  },
                  "range": -1
                }
              ]
            }
          ]
        }
      ]
    }
  ]
}