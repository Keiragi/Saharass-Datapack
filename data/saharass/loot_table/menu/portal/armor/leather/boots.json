{
  "pools": [
    {
      "rolls": 1,
      "entries": [
        {
          "type": "item",
          "name": "leather_boots",
          "functions": [
            {
              "function": "set_name",
              "name": {
                "text": "革のブーツ",
                "color": "white"
              },
              "target": "item_name"
            },
            {
              "function": "set_attributes",
              "modifiers": []
            },
            {
              "function": "set_custom_data",
              "tag": "{\"Menu\":{\"Button\":\"armor/leather/boots\"}}"
            },
            {
              "function": "set_lore",
              "entity": "this",
              "mode": "replace_all",
              "lore": [
                {
                  "text": "保管数 : ",
                  "extra": [
                    {
                      "score": {
                        "objective": "ItemStorage",
                        "name": "$104"
                      },
                      "extra": [
                        {
                          "text": "個"
                        }
                      ]
                    }
                  ],
                  "color": "white",
                  "italic": false
                }
              ]
            },
            {
              "function": "reference",
              "name": "saharass:menu/not_yet",
              "conditions": [
                {
                  "condition": "value_check",
                  "value": {
                    "type": "score",
                    "target": {
                      "type": "fixed",
                      "name": "$104"
                    },
                    "score": "ItemStorage"
                  },
                  "range": -1
                }
              ]
            }
          ]
        }
      ]
    }
  ]
}