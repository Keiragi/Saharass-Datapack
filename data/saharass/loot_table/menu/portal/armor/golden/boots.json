{
  "pools": [
    {
      "rolls": 1,
      "entries": [
        {
          "type": "item",
          "name": "golden_boots",
          "functions": [
            {
              "function": "set_name",
              "name": {
                "text": "金のブーツ",
                "color": "white"
              },
              "target": "item_name"
            },
            {
              "function": "set_attributes",
              "modifiers": []
            },
            {
              "function": "set_custom_data",
              "tag": "{\"Menu\":{\"Button\":\"armor/golden/boots\"}}"
            },
            {
              "function": "set_lore",
              "entity": "this",
              "mode": "replace_all",
              "lore": [
                {
                  "text": "保管数 : ",
                  "extra": [
                    {
                      "score": {
                        "objective": "ItemStorage",
                        "name": "$112"
                      },
                      "extra": [
                        {
                          "text": "個"
                        }
                      ]
                    }
                  ],
                  "color": "white",
                  "italic": false
                }
              ]
            },
            {
              "function": "reference",
              "name": "saharass:menu/not_yet",
              "conditions": [
                {
                  "condition": "value_check",
                  "value": {
                    "type": "score",
                    "target": {
                      "type": "fixed",
                      "name": "$112"
                    },
                    "score": "ItemStorage"
                  },
                  "range": -1
                }
              ]
            }
          ]
        }
      ]
    }
  ]
}